run-build.ps1

```diff
@@ -29,6 +30,10 @@ if ($Sign) {
   $WarnAsError = $false
 }
 
+if ($Pack) {
+  $Parameters = "$Parameters /p:PackInstaller=true"
```

Comment:
You should not need a custom property here. If you pass -pack to build.ps1, `Pack=true` gets passed to all the projects.

In addition, the `SignCoreSdk` property above is dead.

Reply:
With that in mind, run-build.ps1 is looking pretty thin. Aside from Architecture...is it needed at all?

Reply:
@mmitche when I tried using Pack in the conditions, I found that Pack=true gets stripped out after the build.proj evaluation. I tracked that down to this line: https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L114

So after adding the Pack condition, neither of the targets ended up running even though I included -pack. That's why I added PackInstaller as a secondary property.

Reply:
Understood.